### ISSUE TITLE
where is dataframe dataset

### ISSUE THREAD

**Issue by wangjiawen2013** [tracel-ai/burn#2406]
Hi,
The dataframe dataset has already been merged into burn and is demonstrated in burns book:
![image](https://github.com/user-attachments/assets/99e4c664-cbf1-49c5-88cc-c08f1f09b67b)

But I cannot find it in burn's docs and examples, where is it ?

**Comment by HessTaha** [comment]
Hello, 

The code is separated in differents crates.
You Can find the dataset crate under the crates folder:  https://github.com/tracel-ai/burn/tree/main/crates%2Fburn-dataset

**Comment by laggui** [comment]
Right now there are no examples which make use of the `DataframeDataset` (which is under the `dataframe` feature flag for the dataset module).

I agree that it would be nice to have at least a snippet for its usage. I'll convert this issue to track this addition.

**Comment by antimora** [comment]
Yeah it'd be great if we have example. Meanwhile you can see some basic tests:

https://github.com/tracel-ai/burn/blob/main/crates/burn-dataset/src/dataset/dataframe.rs

**Comment by josemanuel657** [comment]
I assume this issue should be closed?

**Comment by laggui** [comment]
Thanks for bringing this up. It has been addressed.
